jinahub/indexers/cache/Dockerfile

```diff
@@ -9,7 +9,6 @@ WORKDIR /workspace
 
 RUN pip install -r requirements.txt
 
-FROM base as test
```

Comment:
Any reason to remove this? If we want to build the test image, we can use
```
docker build --target test -t MyImage .
```

Reply:
adapted all Dockerfiles to be multistage